OrbitGl/ThreadTrack.cpp

```diff
@@ -58,189 +58,134 @@ std::string GetExtraInfo(const Timer& timer) {
   return info;
 }
 
+//-----------------------------------------------------------------------------
+inline Color GetTimerColor(const Timer& timer, TimeGraph* time_graph,
+                           bool is_selected, bool inactive) {
+  const Color kInactiveColor(100, 100, 100, 255);
+  const Color kSelectionColor(0, 128, 255, 255);
+  if (is_selected) {
+    return kSelectionColor;
+  } else if (inactive) {
+    return kInactiveColor;
+  }
+
+  Color color = time_graph->GetTimesliceColor(timer);
+  const uint8_t kOddAlpha = 210;
+  if (!(timer.m_Depth & 0x1)) {
+    color[3] = kOddAlpha;
+  }
+
+  return color;
+}
+
+//-----------------------------------------------------------------------------
+inline float GetYFromDepth(const TimeGraphLayout& layout, float track_y,
+                           uint32_t depth) {
+  return track_y - layout.GetEventTrackHeight() -
+         layout.GetSpaceBetweenTracksAndThread() -
+         layout.GetTextBoxHeight() * (depth + 1);
+}
+
+//-----------------------------------------------------------------------------
+void ThreadTrack::SetTimesliceText(const Timer& timer, double elapsed_us,
+                                   float min_x, TextBox* text_box) {
+  if (text_box->GetText().empty()) {
+    double elapsed_millis = ((double)elapsed_us) * 0.001;
```

Comment:
you probably do not need this cast here - 0.001 should do the job for you

Reply:
true!  done.